var/spack/repos/builtin/packages/rocblas/package.py

```diff
@@ -15,18 +15,23 @@ class Rocblas(CMakePackage):
 
     maintainers = ['haampie']
 
+    version('3.7.0', sha256='9425db5f8e8b6f7fb172d09e2a360025b63a4e54414607709efc5acb28819642')
     version('3.5.0', sha256='8560fabef7f13e8d67da997de2295399f6ec595edfd77e452978c140d5f936f0')
 
     amdgpu_targets = ('all', 'gfx803', 'gfx900', 'gfx906', 'gfx908')
 
     variant('amdgpu_target', default='all', multi=True, values=amdgpu_targets)
 
     depends_on('cmake@3:', type='build')
-    depends_on('rocm-cmake@3.5.0', type='build', when='@3.5.0')
-    depends_on('rocm-device-libs@3.5.0', type='build', when='@3.5.0')
 
-    depends_on('hip@3.5.0', when='@3.5.0')
-    depends_on('comgr@3.5.0', type='build', when='@3.5.0')
+    for ver in ['3.5.0', '3.7.0']:
+        depends_on('rocm-cmake@' + ver, type='build', when='@' + ver)
+        depends_on('rocm-device-libs@' + ver, type='build', when='@' + ver)
+        depends_on('hip@' + ver, when='@' + ver)
+        depends_on('comgr@' + ver, type='build', when='@' + ver)
+
+    # This is the default library format since 3.7.0
+    depends_on('msgpack-c@3:', when='@3.7:')
 
     depends_on('python', type='build')
     depends_on('py-virtualenv', type='build')
```

Comment:
While doing spack install rocblas@3.7.0, i see the below error - 
     8     -- Check for working CXX compiler: /spack/opt/spack/linux-centos7-skylake_avx512/gcc-7.3.1/hip-3.7.0-cdwjw4owlq433ccuoogro
           6ovm7xuafag/bin/hipcc - broken
  >> 9     CMake Error at /spack/opt/spack/linux-centos7-skylake_avx512/gcc-7.3.1/cmake-3.18.2-e73cch6cp6tczpn55v5r2ddxj7ryqdgr/share
           /cmake-3.18/Modules/CMakeTestCXXCompiler.cmake:59 (message):
     10      The C++ compiler
     11
     12        "/spack/opt/spack/linux-centos7-skylake_avx512/gcc-7.3.1/hip-3.7.0-cdwjw4owlq433ccuoogro6ovm7xuafag/bin/hipcc"
     13
     14      is not able to compile a simple test program.
     15

     ...

     21        gmake[1]: Entering directory `/tmp/root/spack-stage/spack-stage-rocblas-3.7.0-6u7ydv5oit5zaqpzlqvy77674cuzhqwu/spack-b
           uild/CMakeFiles/CMakeTmp'
     22        Building CXX object CMakeFiles/cmTC_26937.dir/testCXXCompiler.cxx.o
     23        /spack/opt/spack/linux-centos7-skylake_avx512/gcc-7.3.1/hip-3.7.0-cdwjw4owlq433ccuoogro6ovm7xuafag/bin/hipcc    -o CMa
           keFiles/cmTC_26937.dir/testCXXCompiler.cxx.o -c /tmp/root/spack-stage/spack-stage-rocblas-3.7.0-6u7ydv5oit5zaqpzlqvy77674c
           uzhqwu/spack-build/CMakeFiles/CMakeTmp/testCXXCompiler.cxx
     24        Linking CXX executable cmTC_26937
     25        /spack/opt/spack/linux-centos7-skylake_avx512/gcc-7.3.1/cmake-3.18.2-e73cch6cp6tczpn55v5r2ddxj7ryqdgr/bin/cmake -E cma
           ke_link_script CMakeFiles/cmTC_26937.dir/link.txt --verbose=1
     26        /spack/opt/spack/linux-centos7-skylake_avx512/gcc-7.3.1/hip-3.7.0-cdwjw4owlq433ccuoogro6ovm7xuafag/bin/hipcc CMakeFile
           s/cmTC_26937.dir/testCXXCompiler.cxx.o -o cmTC_26937
  >> 27    /opt/rh/devtoolset-7/root/usr/lib/gcc/x86_64-redhat-linux/7/../../../../bin/ld: cannot find -lnuma
  >> 28        clang-11: error: linker command failed with exit code 1 (use -v to see invocation)
  >> 29        gmake[1]: *** [cmTC_26937] Error 1
     30        gmake[1]: Leaving directory `/tmp/root/spack-stage/spack-stage-rocblas-3.7.0-6u7ydv5oit5zaqpzlqvy77674cuzhqwu/spack-bu
           ild/CMakeFiles/CMakeTmp'
  >> 31        gmake: *** [cmTC_26937/fast] Error 2
     32

Reply:
Hm, in my case I don't get this error, and libnuma is correctly linked:

```
libnuma.so.1 => /apps/ault/SSL/hstoppel/spack/opt/spack/linux-centos7-zen/gcc-10.1.0/numactl-2.0.12-h7sakkcqf4sllxlnl5o4k7jscm6yc6xt/lib/libnuma.so.1 (0x00007faea9736000)
```

But probably we have to make this explicit